nucleus/async_job.py

```diff
@@ -131,6 +131,27 @@ def from_json(cls, payload: dict, client):
         )
 
 
+class EmbeddingsExportJob(AsyncJob):
+    def result_urls(self) -> List[str]:
```

Comment:
I'm wondering how you would instantiate this in another process. I think we need a classmethod `from_id` that would allow you to spin this up in one environment and then poll in another just from the `job_id`

Reply:
Good point, that would be used through the `NucleusClient.list_jobs` method though right? So something like this:

```
jobs = NucleusClient.list_jobs()
export_job = EmbeddingsExportJob.from_job_id(jobs[0].job_id)
```